### PR TITLE
feat theme toggle

### PR DESCRIPTION
**Pull request by hsnice16**
#### This PR will add code for the dark mode theme.

This is how it looks,
<video src="https://user-images.githubusercontent.com/56081584/155154093-2fa2361b-40c7-4c9b-b9d0-4005c8715b7d.mp4"></video>

#### Please review this PR, and give your feedback.

### CODE REVIEW

**Review by vaishnavipaliya11 — COMMENTED** [review]
Please don't make changes to so many files in a single PR.

**Comment by hsnice16** [comment]
> Please don't make changes to so many files in a single PR.

The changes are the same lines of code, there are files for each question therefore same lines of code are present in many files.

**Review by vrx29 — APPROVED** [review]
Nice work Himanshu. you have used a dark theme that looks really good.

File: app.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    localStorage.setItem("marvelsQuizTheme", "dark");
+  } else {
+    htmlElement.removeAttribute("data-theme");
+    localStorage.removeItem("marvelsQuizTheme");
+  }
+
+  moonIcon.classList.toggle("fas");
+});
+
+(() => {
```
Comment (jalajcodes) [inline app.js:17]: why is this inside an IIFE? it will work without it

**Review by jalajcodes — COMMENTED** [review]
Nice work, just a small query...

**Comment by jalajcodes** [comment]
> > Please don't make changes to so many files in a single PR.
> 
> The changes are the same lines of code, there are files for each question therefore same lines of code are present in many files.

Mention this in the PR so people don't get afraid😂

File: app.js (hunk 1 above)
Comment (hsnice16) [reply app.js:17]: As, I am storing the chosen theme, that the user chooses, in the local storage. So when the user comes back to the site, or moves to some other page, I am checking if it has a preferred dark theme, then do changes accordingly.

And as this has to be known as the page loads, and only one time, that's why I have used IIFE.

**Comment by hsnice16** [comment]
Thank you for reviewing this.

**Pull request merged**